lib/datadog/statsd.rb

```diff
@@ -358,10 +358,11 @@ def escape_event_content(msg)
     end
 
     def escape_tag_content(tag)
-      remove_pipes(tag).gsub COMMA, BLANK
+      remove_pipes(tag.to_s).gsub COMMA, BLANK
     end
 
     def escape_tag_content!(tag)
+      tag = tag.to_s
```

Comment:
I don't think we need to check for those, the documentation clearly specifies that tags should be an array of strings. I'm ok with supporting tags because it used to work with 1.x and the change is small, but people reading the documentation will/should likely use strings.

Actually, I think this change is not needed with https://github.com/DataDog/dogstatsd-ruby/pull/51.